src/main/java/frc/robot/commands/ResetRamp.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.subsystems.Ramp;
+
+public class ResetRamp extends Command {
+  /** Creates a new ResetRamp. */
+  private Ramp ramp; 
+  private double startTime;
+
+  public ResetRamp(Ramp ramp) {
+    this.ramp = ramp;
+    addRequirements(ramp);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    startTime = Timer.getFPGATimestamp();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    ramp.changeRampPos(99);
```

Comment:
This seems like a strange way to do this. First, how do you know which way is "forward"? Should this 99 be a constant? Why 99? Why not 9999?

Reply:
we're using motor speed now, still assuming a lot